"Cash by Mail" instructions improvements

Jakub-CZ opened the pull request:
<!-- 
- make yourself familiar with the CONTRIBUTING.md if you have not already (https://github.com/bisq-network/bisq/blob/master/CONTRIBUTING.md)
- make sure you follow our [coding style guidelines][https://github.com/bisq-network/style/issues)
- pick a descriptive title
- provide some meaningful PR description below
- create the PR
- in case you receive a "Change request" and/or a NACK, please react within 30 days. If not, we will close your PR and it can not be up for compensation.
- After addressing the change request, __please re-request a review!__ Otherwise we might miss your PR as we tend to only look at pull requests tagged with a "review required".


Fixes #replaceWithIssueNr, fixes #replaceWithIssueNr
-->
### 1) Fix spaces in "Cash by Mail"
In the English string `portfolio.pending.step2_buyer.cashByMail` a space is missing after the 2nd sentence.

#### Screenshot in Transifex:
![image](https://user-images.githubusercontent.com/16313562/105611930-d131ce80-5db8-11eb-91c1-f8c23a25537e.png)

### 2) Clarify the process of mediation
@pazza83's suggestion from https://github.com/bisq-network/bisq/pull/5092#issuecomment-765730782

![image](https://user-images.githubusercontent.com/16313562/105616485-8d9a8d00-5dd7-11eb-896b-e9babc695401.png)

Jakub-CZ committed:
Fix spaces in "Cash by Mail"

ghost commented:
Hopefully when the revised trading rules text is ready, @Jakub-CZ will include it in this PR.
Relevant discussion of the proposed change: https://github.com/bisq-network/bisq/pull/5092#issuecomment-765053669 and onward.

Jakub-CZ committed:
Add details about mediation in "Cash by mail"

Jakub-CZ committed:
Fix wording

Jakub-CZ committed:
Resolve unintended ellipsis. Make popup wider.

Jakub-CZ commented:
As suggested by @jmacxx, I've implemented @pazza83's suggestion from #5092 (after the PR got merged).

### Additional changes:
- fixed wording of "... could lead to lose both trading and deposit funds" (in the 5th bullet point in the second list; please somebody review);
- made the popup wider for fewer line breaks;
- bumped up the hardcoded character limit to avoid ellipsis in this long block of text.

![image](https://user-images.githubusercontent.com/16313562/105616485-8d9a8d00-5dd7-11eb-896b-e9babc695401.png)

ripcurlx reviewed (approved):
utACK
